### PR TITLE
Add inspiral-range

### PR DESCRIPTION
**Pull request by duncanmmacleod**
This PR adds a recipe for the `inspiral-range` (`inspiral_range` on pypi) package, used in gravitational-wave astrophysics.

Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by duncanmmacleod** [comment]
@jrollins, please confirm here that you are willing to be listed as a co-maintainer of this recipe.

File: recipes/inspiral-range/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    - pip
+    - python
+  run:
+    - astropy
+    - python
+    - scipy
+
+test:
+  requires:
+    - python-lal  # [not win]
```
Comment (xhochy) [inline recipes/inspiral-range/meta.yaml:31]: If it is required for the tests, why isn't it a `run` dependency?

Comment (duncanmmacleod) [reply recipes/inspiral-range/meta.yaml:31]: `python-lal` is an optional dependency, and having provides a more robust test, but it isn't available on windows, so it shouldn't be a full dependency.

**Comment by jrollins** [comment]
> @jrollins, please confirm here that you are willing to be listed as a co-maintainer of this recipe.

Yes I confirm that I am willing to co-maintain.  Thank you @duncanmmacleod

File: recipes/inspiral-range/meta.yaml (hunk 1 above)
Comment (jrollins) [reply recipes/inspiral-range/meta.yaml:31]: python-lal is not actually required for the tests either.

But python-lal should be preferred as a runtime/test dependency over python-astropy, if it's possible to express that somehow...

**Comment by jrollins** [comment]
@duncanmmacleod I pushed a 0.3.1 that fixes the lal import issue.

**Comment by duncanmmacleod** [comment]
@conda-forge/staged-recipes, this recipe is ready for review, thanks.

**Pull request merged**